db/migrate/20150824155143_r3_media_object_to_r4.rb

```diff
@@ -0,0 +1,27 @@
+class R3MediaObjectToR4 < ActiveRecord::Migration
+  def up
+    say_with_time("R3->R4") do
+      prefix = Avalon::Configuration.lookup('fedora.namespace')
+      ActiveFedora::Base.reindex_everything("pid~#{prefix}:*")
```

Comment:
I'm not sure. I'll guess I was just following the example of a previous migration.